e2e/chain/e2e_test.go

```diff
@@ -65,3 +69,23 @@ func TestTransactionStoredOnChain(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, len(txs.Epochs) != 0)
 }
+
+// TestNamespaceSaved ensures that when the stack is running, namespaces can be queried
+// and the address in the namespace can be supplied to a router msg client and send requests.
+func TestNamespaceSaved(t *testing.T) {
+	t.Skip("do not run this test unless the docker containers for base shard, game shard, nakama, and " +
+		"celestia are running. to run the stack, run `make game` and `make rollup`. wait for startup and run this test")
+	chain := newChainClient(t)
+	res, err := chain.namespace.Namespaces(context.Background(), &namespacetypes.NamespacesRequest{})
+	assert.NilError(t, err)
+	assert.Equal(t, len(res.Namespaces), 1)
```

Comment:
I'm seeing this failure when I run it locally:

```
=== RUN   TestNamespaceSaved
    e2e_test.go:83: assertion failed: 0 (int) != 1 (int)
```

Reply:
updated the `make` command needed to run the game stack properly.

Reply:
👍 test is passing with the updated instructions.